style.css

```diff
@@ -750,3 +756,370 @@ ul {
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger,
+  .ambony {
+    display: none;
+  }
```

Comment:
- [ ] Good job hiding the menu icon but it seems like the navigation links are hidden as well so please make sure they're visible as they'll help the user easily navigate through your portfolio providing a better UX.
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230614891-7e7bd8fe-640d-4129-8403-40bd84e4df3c.png)
     - Figma design:
       ![image](https://user-images.githubusercontent.com/104968236/230615042-f19fa628-0ef8-4fa0-9fe1-698a3c9fae17.png)
- [ ] Kindly make sure there's some spacing to the left of the logo and to the right of your nav links as specified in the design🎨.